### ISSUE TITLE
Exception thrown: 'Dicom.DicomDataException' in Dicom.Core.dll: 'UID: Unknown [1.2.840.10008.1.2.2.123456] is not a transfer syntax type.'    at Dicom.DicomTransferSyntax.Lookup(DicomUID uid)  ...

### ISSUE THREAD

**Issue by buckstephenh** [fo-dicom/fo-dicom#703]
### Expected behavior

Round trip of DicomTransferSyntax elements to DicomTransferSyntax.Parse(uid) should be consistent.

### Actual behavior

Saving DicomTransferSyntax elements and calling DicomTransferSyntax.Parse(uid) with the number indicates 1.2.840.10008.1.2.2.123456 is not known, yet it came directly from "Implicit VR Big Endian [1.2.840.10008.1.2.2.123456]" ie by serializing public static readonly DicomTransferSyntax ImplicitVRBigEndian;  

Other parses work correctly so maybe the 123456 is the problem? 

### Steps to reproduce the behavior

var transfersyntax = DicomTransferSyntax.Parse(1.2.840.10008.1.2.2.123456);

### fo-dicom version and OS/platform
Mac: fo-dicom" Version="4.0.0-rc1"

**Comment by gofal** [comment]
This ImplicitBitEndian is not a valid transfer syntax, so the exception seems to be fine.
If you take a look at the definiton of the DicomTransferSytax.ImplicitBigEndian:

```
        /// <summary>Virtual transfer syntax for reading datasets improperly encoded in Big Endian format with implicit VR.</summary>
        public static readonly DicomTransferSyntax ImplicitVRBigEndian =
            new DicomTransferSyntax(new DicomUID(DicomUID.ExplicitVRBigEndianRETIRED.UID + ".123456",
                "Implicit VR Big Endian", DicomUidType.TransferSyntax))
            {
                IsExplicitVR = false,
                Endian = Endian.Big
            };
```

this is only a virtual and not valid transfersyntax. If you take a look into DICOM standard, you will not find any Implicit Big Endian.

So the question is rather, why is there this virtual definition in DicomTransferSyntax. It was added 11 months ago by @anders9ustafsson . Maybe he could tell us the reason.

**Comment by buckstephenh** [comment]
Here's an interesting find of a vendor that cites 
1.2.840.113619.5.2 | Implicit VR - Big Endian (G.E Private) ( http://www.softneta.com/files/meddreamviewer/50/conformance-5.0.pdf )

I am guessing there can be any number of Private transfer syntaxes?

**Comment by gofal** [comment]
I see. So what is your suggestion how to handle/solve this issue? Because only not-throwing would not be enought, the private transfer syntax then has to be handled correctly, hasn't it?

**Comment by gofal** [comment]
fixed with #753
